.circleci/config.yml

```diff
@@ -3,12 +3,140 @@ version: 2.1
 orbs:
   prodsec: snyk/prodsec-orb@1.0
 
+defaults: &defaults
+  parameters:
+    node_version:
+      type: string
+      default: ""
+  working_directory: ~/resolve-deps
+
+commands:
+  install_deps:
+    description: Install dependencies
+    steps:
+      - checkout
+      - run:
+          name: Use snyk-main npmjs user
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+      - run:
+          name: Install dependencies
+          command: npm install
+  install_node_npm:
+    description: Install correct Node version
+    parameters:
+      node_version:
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: Install correct version of Node
+          command: nvm install << parameters.node_version >>
+      - run:
+          name: Use correct version of Node
+          command: nvm use << parameters.node_version >>
+  show_node_version:
+    description: Log Node and npm version
+    steps:
+      - run:
+          name: Node version
+          command: node --version
+      - run:
+          name: NPM version
+          command: npm --version
+
+jobs:
+  lint:
+    <<: *defaults
+    docker:
+      - image: cimg/node:<< parameters.node_version >>
+    steps:
+      - show_node_version
+      - checkout
+      - attach_workspace:
+          at: ~/resolve-deps
+      - install_deps
+      - run:
+          name: Run lint
+          command: npm run lint
+
+  test-unix:
+    <<: *defaults
+    docker:
+      - image: cimg/node:<< parameters.node_version >>
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/resolve-deps
+      - install_deps
+      - show_node_version
+      - run:
+          name: Run tests
+          command: npm test
+
+  release:
+    <<: *defaults
+    docker:
+      - image: cimg/node:<< parameters.node_version >>
+    resource_class: small
+    steps:
+      - install_deps
+      - run: sudo npm i -g semantic-release@17 @semantic-release/exec pkg
+      - run:
+          name: Publish to GitHub
+          command: semantic-release
```

Comment:
I think I remember somewhere that if/when we switch default branch to `main` we need a `.releaserc` too.

Reply:
Maybe we monorepo by then just so we have one large CI and we don't get drift like this :)